packages/api/cms-api/src/user-permissions/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Type } from "@nestjs/common";
+
+import { AuthProxyJwtStrategyConfig, createAuthProxyJwtStrategy } from "../../auth/strategies/auth-proxy-jwt.strategy";
+import { CurrentUser } from "../dto/current-user";
+import { UserPermissionsService } from "../user-permissions.service";
+
+export function createUserPermissionsJwtStrategy(config: AuthProxyJwtStrategyConfig): Type {
```

Comment:
Why do we need this new strategies? Can't we adapt the existing ones?

Reply:
The alternative would be just to use a UserPermissions-module specific UserLoader. However, this would also require a rewrite of the current Strategies to be able to use DI. We already had a discussion like this in https://github.com/vivid-planet/comet/pull/995 an decided to stick with the Factories because in the end it doesn't make much of a difference.